source/install/desktop.rst

```diff
@@ -57,11 +57,13 @@ The following Group Policies are available:
 macOS 10.9+
 --------------------------------------------------
 
-1. Download `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-mac.dmg>`__.
+1. Download the `latest version of the Mattermost desktop app:
+
+ - <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-mac.dmg>`__
```

Comment:
The link doesn't appear correctly if it's separated from the line above.

Reply:
Didn't catch that one, thanks. It's fixed.